src/style.css

```
@@ -0,0 +1,28 @@
+* {
+  box-sizing: border-box;
+}
+
+body {
+  background-color: #000;
+  margin: 0;
+  padding: 20px;
+}
+
+.redBorder {
+  padding: 10px;
+  border: #e0524e 1px solid;
```

дававй назви класів писати через `-`

прийнято записувати в такий послідовності
```suggestion
  border: 1px solid #e0524e;
```

---

название класса